jenkins/jenkinsfile

```diff
@@ -3,7 +3,7 @@ pipeline {
          docker {
             label 'AL2-X64'
             /* See  
-            https://github.com/opensearch-project/opensearch-build/blob/main/docker/ci/dockerfiles/build.ubuntu18.opensearch.x64.dockerfile
+            https://hub.docker.com/layers/ci-runner/opensearchstaging/ci-runner/ci-runner-ubuntu[…]3c382ac53db414754dabd52f9b70e1c7e344dfb9a0e5e?context=explore
```

Comment:
Is the link accessible by the public?

Reply:
Thanks for pointing it out. A small change in the url was not letting it accessible. Pushed the change.

Reply:
😁 The link can be accessed now.